### ISSUE TITLE
Option to hide .bsky.social portion of user handle

### ISSUE THREAD

**Issue by hopqer** [bluesky-social/social-app#5424]
Some users choose a custom domain as their bluesky handle, and that's great, especially for verification purposes! But for those who just want to remain unverified & local with username dot bsky dot social, it would look super clean if given the option - at least in the feed - to hide the dot bsky dot social portion of the URL. (Though it would also vastly improve mentions by not eating up as much of the character limit, I'm not sure how that might affect user tagging on the back end.)

So for example:  (@)hopper.bsky.social would just appear as (@)hopper in the feed while (@)hopper(.)com would show the entire URL as the handle

**Comment by AdrianoCahete** [comment]
I don't think that this will happen because the way federate id works. You can have a lot of PDS'es, without a center Auth server. 

But, I think that they can change how the handle is displayed on profile, and, maybe, on messages/citations. On profile they have more flexibility, but on messages is a little different, small window of improvement in there, IMO.

Who is the UI/UX responsible for project, @pfrazee @mozzius? (sorry for always tag you two, since I'm Brazilian, is the only 2 that I easier remember Hahahahah)

**Comment by electricduck** [comment]
This is something Mastodon did and, honestly, it made everything far more confusing. Once they made the change I saw more and more people just doing `@username` in posts (instead of `@username@server.com`) which, of course, didn't actually mention anyone. The domain suffix is a **first-class important part of the username** and there is simply no benefit of hiding it.

**Comment by Yolwoocle** [comment]
Perhaps it could be grayed out so it stands out more, but agreed with others; the .bsky.social part is a very important part of the username and hiding it could just lead to confusion.

**Comment by HorusGoul** [comment]
The AT Protocol discourages doing this, check out the docs:

> Handles should generally not be truncated to local context. For example, the handle `@jay.bsky.social` should not be displayed as `@jay`, even in the local context of a `bsky.social` service.

https://atproto.com/specs/handle#:~:text=Handles%20should%20generally%20not%20be%20truncated%20to%20local%20context.%20For%20example%2C%20the%20handle%20%40jay.bsky.social%20should%20not%20be%20displayed%20as%20%40jay%2C%20even%20in%20the%20local%20context%20of%20a%20bsky.social%20service.

**Comment by mopoIo** [comment]
If you're on desktop you can[ try this script I just made](https://github.com/mopoIo/bluesky-handle-cleaner)
